readme.md

```diff
@@ -184,8 +190,7 @@ Then, go to [http://localhost:5000/](http://localhost:5000) and you should see a
 * **Question:** The above can be described as middleware. Describe what middleware is in your own words and how it differs from building a route.
 
 * **Your Answer:**
-
----
+* The middleware will run before a route. 
```

Comment:
Sometimes, depends on where it is defined in the route chain. 
Middleware is the code/application/module that communicates between two systems, such as between servers and databases. The code above is different than a route because it is always run whenever the application gets to these lines.